data.sql

```diff
@@ -47,3 +47,181 @@ INSERT INTO animals (name, species_id, owner_id) VALUES
 ('Angemon', 2, 5),
 ('Boarmon', 1, 5);
 
+-- Modify inserted animals so it includes the species_id value:
+-- If the name ends in "mon" it will be Digimon
+-- All other animals are Pokemon
+BEGIN;
+UPDATE animals
+SET species_id = (SELECT id FROM species WHERE name = 'Digimon')
+WHERE name like '%mon';
+UPDATE animals
+SET species_id = (SELECT id FROM species WHERE name = 'Pokemon')
+WHERE species_id is null;
+COMMIT;
+SELECT * from aniSmals;
+
+-- Modify inserted animals to include owner information (owner_id)
+SELECT * FROM owners;
+BEGIN;
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith')
+WHERE name like 'Agumon';
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell')
+WHERE name IN ('Gabumon', 'Pikachu');
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob')
+WHERE name IN ('Devimon', 'Plantmon');
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond')
+WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
+WHERE name IN ('Angemon', 'Boarmon');
+COMMIT;
+SELECT * from animals;
+ROLLBACK;
+
```

Comment:
- [ ] Please make sure that everything that is a query that helps to analyze data should be saved in the `queries.sql` file. :pray:

Reply:
@Owusu-Desmond Thanks a lot for your kind review! All requested changes implemented. Cheers!